option optimize_for = LITE_RUNTIME;
import "U64Type.proto";
message FriendOperatorAnswer
{
 required uint32 result = 1;
 repeated U64Type playerid = 2;
}
